lib/tools/adb-commands.js

```diff
@@ -913,13 +915,14 @@ methods.getGeoLocation = async function () {
       `services must be enabled on the device. Original error: ${err.message}`);
   }
 
-  const match = /data="(-?[\d\.]+)\s+(-?[\d\.]+)"/.exec(output);
+  const match = /data="(-?[\d\.]+)\s+(-?[\d\.]+)\s+(-?[\d\.]+)"/.exec(output);
```

Comment:
does this regex allow for the third number to be absent in the case of running an older appium settings?

Reply:
no it does not, but I assume there will by anyway the minimum package version set in android driver, so we don't have to care much about it

Reply:
ok, as long as android driver performs an upgrade on the settings app if necessary 👍

Reply:
yes it does
https://github.com/appium/appium-android-driver/blob/05e7a982f4c44c7bd09922caf02079d3e561681c/lib/android-helpers.js#L438